draft-ietf-ohai-ohttp.md

```diff
@@ -1263,8 +1270,9 @@ payload is appropriately reflected in the HPKE info and context strings. For
 example, if a future specification were to use the encryption mechanism in
 this specification for DNS messages, identified by the "application/dns-message"
 media type, then the HPKE info string SHOULD be "application/dns-message
-request" for request encryption, and the HPKE export context string should be
-"application/dns-message response" for response encryption.
+request", a zero byte, and the header for request encryption. Similarly,
```

Comment:
This is the one "bug" in the specification I saw, where we didn't update the info string suggestion